docs/testing_your_changes.md

```diff
@@ -0,0 +1,52 @@
+## Location of tests
+
+OpenFGA strives to maintain good code coverage with tests so that the possibility of regressions is reduced.
+
+The purpose of this guide is to inform contributors where to write new tests.
+
+## 1. Storage layer
+
+1. Any change to the OpenFGA storage interface must have an integration test in pkg/storage/test/storage.go. This is so that anyone writing their own storage adapter can have confidence that it is correct.
```

Comment:
```suggestion
1. Any change to the OpenFGA storage interface must have an integration test in `pkg/storage/test/storage.go`. This is so that anyone writing their own storage adapter can have confidence that it is correct.
```